### ISSUE TITLE
[Bug]: ipcRenderer.invoke now sends a weird object

### ISSUE THREAD

**Issue by DANser-freelancer** [electron/electron#43625]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

31.5.0

### What operating system(s) are you using?

Windows

### Operating System Version

11

### What arch are you using?

x64

### Expected Behavior

I expect my ipcRenderer.invoke() calls to pass 3 params.

### Actual Behavior

Instead it passes 3 params **after** an object:        
```            
{"_replyChannel":{},"frameId":1,"processId":4,"sender":{"ipc":{"_events":{},"_eventsCount":1,"_invokeHandlers":{}},"navigationHistory":{},"_events":{},"_eventsCount":16}}               
```         
the properties are underscored so I assume this should be private.

### Additional Information

Just to clarify, I reinstalled the whole setup and only moved in the files I personally modified, I have an untouched electron-vite setup with only main, preload, and renderer modified. Reinstalling did not fix anything.          
I have this whole reddit post describing weird stuff going on https://www.reddit.com/r/electronjs/comments/1fc6d1w/something_is_broken_and_i_have_no_idea_what/             
Basically a couple days ago everything worked fine but now there's a strange object being sent.

**Comment by DominikDoom** [comment]
Not a bug. I would suggest reading the documentation first next time:
https://www.electronjs.org/docs/latest/api/ipc-main#ipcmainhandlechannel-listener
As is clearly stated there, the first argument of the listener function is an [IpcMainInvokeEvent](https://www.electronjs.org/docs/latest/api/structures/ipc-main-invoke-event), similarly to every other event callback.

The same info is also written in the type definitions, so this is not something hidden in any way:
![image](https://github.com/user-attachments/assets/13b2eaa4-5855-430f-80e0-c8ea60c21bfd)
At least for VSCode, this should also be shown if you use plain JS.

If you use a linter that complains about the event parameter being unused in your listener, you can prefix the name with an underscore (e.g. `_event`) or even use just an underscore, which specifies you don't want to use it.

Also, this is not something new. If you didn't experience this issue before, it was likely just because your listener either used no args at all or consumed the event object instead of your actual value without you noticing.

**Comment by DANser-freelancer** [comment]
For the record, it worked perfectly fine somehow before. And I tried to read docs but apparently you have more docs than my neighbor potted plants (trust me, she has an entire jungle).
